pkg/tempo/config.go

```diff
@@ -90,9 +97,10 @@ type PushConfig struct {
 	Insecure           bool                   `yaml:"insecure"`
 	InsecureSkipVerify bool                   `yaml:"insecure_skip_verify"`
 	BasicAuth          *prom_config.BasicAuth `yaml:"basic_auth,omitempty"`
-	Batch              map[string]interface{} `yaml:"batch,omitempty"`            // https://github.com/open-telemetry/opentelemetry-collector/blob/7d7ae2eb34b5d387627875c498d7f43619f37ee3/processor/batchprocessor/config.go#L24
```

Comment:
Since PushConfig and []RemoteWriteConfigs are mutually exclusive, I think we should leave Batch in the PushConfig for backwards compat?

Either the user configures the old way which doesn't allow split pipelines and the BatchConfig can be directly in the push config or they configure the new way which does allow split pipelines and the BatchConfig has to be on the InstanceConfig.

Reply:
Yes, you're right. Addressed in 7dcc416f428e935f28f824e99973b46c952f4255